content/blog/2019-12-03-mark-padgham.md

```diff
@@ -0,0 +1,76 @@
+
+Mark has made many contributions to rOpenSci prior to joining the team. He has developed four peer-reviewed packages: [osmdata](https://docs.ropensci.org/osmdata/) to import Open Street Map data[^1]; [osmplotr](https://docs.ropensci.org/osmplotr/) to make bespoke plots with that data; [bikedata](https://docs.ropensci.org/bikedata/) to access historical trip data from public bicycle-hire systems [^2], and most recently, [mapscanner](https://github.com/ropensci/software-review/issues/330) to print maps, draw on them, and scan them back in. Mark also contributed to the development of [stats19](https://docs.ropensci.org/stats19/)[^3]. He has reviewed three packages: [piggyback](https://github.com/ropensci/software-review/issues/220), [tradestatistics](https://github.com/ropensci/software-review/issues/274), and [bomrang](https://github.com/ropensci/software-review/issues/121)[^4], authored [two blog posts](/authors/mark-padgham/), and participated in rOpenSci [unconf18](/blog/2018/06/05/unconf18/).
+
+_Note from Stefanie:_ When I met Mark in Seattle at rOpenSci’s unconf18, I was struck by his personal commitment to “walking the talk” of getting around by bicycle. I knew him only from his blog post on [getting data from public bicycle rental systems](/blog/2017/10/17/bikedata/), and there he was with colleagues, walking down the street with a brightly-colored rental bike. He said he made a point of using such clearly marked rental bikes when visiting other cities since they’re great conversation-starters about sustainable cities.
+
+{{< figure src = "/img/team/mark-padgham.jpg" width = "250" alt = "Headshot of Mark Padgham" >}}
+
+> I’ve got a lot of concrete visions for extending testing well beyond anything that’s been done in a computational or quantitative way. Convincing leaders in statistics and statistical software that’s a good idea is certainly going to be challenging and very rewarding.
+
+We talked with Mark to learn more about him, and his work.
+
+
+### What do you want people to know about your background?
+
+I live in Germany, but grew up in Australia. I have an undergraduate degree in physics and maths, a PhD in ecology, qualifications in climatology and meteorology, as well as official teaching qualifications. All of these were obtained from universities in Melbourne, Australia - each one from a different university. Following my PhD, I worked in post-doctoral academic positions at the interface of ecology and climatology, before changing country and continent to Germany, and changing academic direction to the study of urban systems.
```

Comment:
"I am originally from Australia, have lived and worked in a few countries along the way (including Japan and Austria), and now live in Germany."